autopilot/interface.go

```diff
@@ -211,14 +211,4 @@ type ChannelController interface {
 	//
 	// TODO(roasbeef): add force option?
 	CloseChannel(chanPoint *wire.OutPoint) error
-
-	// SpliceIn attempts to add additional funds to the target channel via
-	// a splice in mechanism. The new channel with an updated capacity
-	// should be returned.
-	SpliceIn(chanPoint *wire.OutPoint, amt btcutil.Amount) (*Channel, error)
```

Comment:
Just a taddd premature 😂

Reply:
I could also implement it in this PR if that is preferred?

Reply:
Please :slightly_smiling_face: